tests/storage/test_storage_api.py

```diff
@@ -90,3 +90,34 @@ def test_observation_attributes(populated_db):
 
     with StorageApi(rdb_url=populated_db, blob_url=populated_db) as api:
         assert api.get_observation_attribute(name, attr) == expected
+
+
+def test_single_observation_misfit_calculation(populated_db):
+    # observation
+    values_obs = [10.1, 10.2]
+    stds_obs = [1, 3]
+    data_indexes_obs = [2, 3]
+    # response
+    values_res = [11.1, 11.2, 9.9, 9.3]
+
+    misfit_manual = {
```

Comment:
Would prefer if this was called misfit_expected, but let's move it to the cleanup issue